modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In the console, select *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
```

Comment:
This is truly nitpicky, but per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#usage-and-highlighting-for-interface-elements), I feel like "click" might be the more appropriate verb here, if these UI elements could be considered either menus or tabs:
```suggestion
. In the console, click *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
```